src/ResourceManager/Relay/Commands.Relay/Cmdlets/HybridConnections/RemoveAzureRelayHybridConnectionsAuthorizationRules.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Relay.Commands.HybridConnections
+{
+    /// <summary>
+    /// 'Remove-AzureRmRelayHybridConnectionAuthorizationRule' Cmdlet removes/deletes AuthorizationRule
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, RelayHybridConnectionAuthorizationRuleVerb, SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveAzureRelayHybridConnectionsAuthorizationRule : AzureRelayCmdletBase
```

Comment:
Again, I don't think we need this cmdlet, right, we should have  single new, get, remove-AzuthroizatinRule cmdlets